keycloak/keycloak_client.go

```diff
@@ -280,6 +281,7 @@ func (keycloakClient *KeycloakClient) sendRequest(request *http.Request) ([]byte
 	}
 
 	if response.StatusCode >= 400 {
+		fmt.Println(string(body), "Error with the response")
```

Comment:
We're already printing the body on line 280, is there a reason this is needed?

In case you weren't aware, you can set the environment variable `TF_LOG` to `DEBUG` to enable debug logging which should print out a bunch of http request/response information.